lib/init.coffee

```diff
@@ -1,8 +1,50 @@
+{BufferedProcess, CompositeDisposable} = require 'atom'
+
 module.exports =
+
   config:
     javaExecutablePath:
       type: 'string'
-      default: ''
+      title: 'Path to the javac executable'
+      default: '/usr/bin/javac'
```

Comment:
This should be changed to `javac` which will auto-resolve to `/usr/bin/javac` in 90% of cases. In the other 10% it will resolve to something else.